frontend/packages/ceph-storage-plugin/src/plugin.ts

```diff
@@ -653,6 +655,31 @@ const plugin: Plugin<ConsumedExtensions> = [
       required: [NOOBAA_FLAG],
     },
   },
+  {
+    type: 'NavItem/Href',
+    properties: {
+      id: 'ocsdashboards',
+      section: 'storage',
+      insertBefore: 'networkpolicies',
```

Comment:
```suggestion
      insertBefore: 'persistentvolumes,
```

Reply:
Won't work with "persistentvolumes"... it is referring the resource by its "id" (check `admin-nav.tsx --> NavSection (Storage) --> Resource (persistentvolumes)`; "id" used here is  "networkpolicies") https://github.com/openshift/console/blob/8b9c03c10635f3ea310b74deeb0576c56359e735/frontend/public/components/nav/admin-nav.tsx#L211
Should I change the "id" corresponding to "persistentvolumes" in admin-nav.tsx page??

Reply:
oh okay, wow yeah please change it there as well. It's a mistake.